Formula/gq-cli.rb

```diff
@@ -0,0 +1,20 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
+  homepage "https://goquant.io"
+  url "https://brickelldatabros1aboah.blob.core.windows.net/public/gq-cli-source-0.0.4.tar.gz"
+  sha256 "6e2b86cf57035bba045d7db9f1d28ae83283cec6b0ed4335083f320374b951ba"
+  license "MIT"
+
+  depends_on "rust" => :build
```

Comment:
Its dependency `native-tls` [depends on OpenSSL on Linux](https://github.com/sfackler/rust-native-tls/blob/master/Cargo.toml).

```suggestion
  depends_on "rust" => :build

  on_linux do
    depends_on "pkg-config" => :build
    depends_on "openssl@3"
  end
```

Reply:
Ok that is very helpful, I was wondering why that didn't come up on our internal pipeline.

Reply:
Probably because OpenSSL is very commonly installed.